pyproject.toml

```diff
@@ -26,7 +26,10 @@ dependencies = [
   "pandas",
   "geopy",
   "tqdm",
-  "python-dotenv"
+  "python-dotenv", 
+  "os", 
```

Comment:
Our "Install Dependencies" step in the Code Quality workflow is failing because of this: https://github.com/AmericanRedCross/street-view-green-view/actions/runs/8442740425/job/23222294773?pr=29#step:4:50